### PR TITLE
[WIP] added README for MNIST

### PR DESCRIPTION
**Pull request by codeboy5**
I have added a README for the dataset and how to process it to be used in the models.
@kartikdutt18 Kindly go through this and let me know what can I improve on.
As discussed in #65

### CODE REVIEW

File: Kaggle/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# MNIST
+
+### DESCRIPTION OF THE DATASET
+
+MNIST dataset contains images of handwritten digits. It is one of the most common datasets used for image classifcation. It has 60,000 28x28 grayscale images under the training set and 10,000 28x28 grayscale images under the test set. Each pixel has a value between 0 and 255.
+
+![alt text](https://upload.wikimedia.org/wikipedia/commons/2/27/MnistExamples.png "Sample Images From The MNIST Database")
+*Sample Images From The MNIST Database*
```
Comment (kartikdutt18) [inline Kaggle/README.md]: Could we have this in a separate line ?
Also The images might look better with centre alignment. Is there any way to do that?

File: Kaggle/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+##### 2. Splitting The Dataset
+The Split Class is used to split the Dataset into Train and Validation Datasets. The train dataset is used in the training procedure, the weights are updated and adjusted to fit this dataset. The validation dataset is used to constantly evaludate the model, which is used to adjust the model hyperparamters like the learning rate. The data is split randomly and RATIO defines the amount of data in Validation Set.
+After splitting the train dataset is stored in the *train* matrix and validation in the *valid* matrix.
+
+### Results Obtained on Different Models
+
+##### 1. DigitRecognizer
+For More Details related to the Model, check out the **DigitRecognizer** Directory.
+After running the model for 50 cycles, I was able to obtain the best validation set accuracy of **97.2143%**.
```
Comment (kartikdutt18) [inline Kaggle/README.md]: Instead of using `I` here can we use `one` here?

File: Kaggle/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+![alt text](https://upload.wikimedia.org/wikipedia/commons/2/27/MnistExamples.png "Sample Images From The MNIST Database")
+*Sample Images From The MNIST Database*
+
+![alt text](https://miro.medium.com/max/490/1*nlfLUgHUEj5vW7WVJpxY-g.png "A visualization of the sample image at index 7777")
+*A visualization of the sample image at index 7777*
+
+### Processing The Dataset
+
+##### 1. Loading The Dataset From The File To A Useable Format
+The Load Class is used to load a matrix from the file.The type of file is guessed automatically from the extension. For a list of supported file types please refer to [Load Class](https://github.com/mlpack/mlpack/blob/master/src/mlpack/core/data/load.hpp). Please note that since mlpack requires matrices to be in column major format and usually data is stored in a row major format, the matrix by default is transposed. In case the data is originally in column major format, consider setting the parameter to False. Refer to documentation of Load Class for further information.
```
Comment (kartikdutt18) [inline Kaggle/README.md]: Do this we could add a code snippet here that shows this function loading mnist.

Comment (kartikdutt18) [reply Kaggle/README.md]: Same for below.

File: Kaggle/README.md (hunk 1 above)
Comment (codeboy5) [reply Kaggle/README.md]: Yeah, I found a way online, would try it and see.

File: Kaggle/README.md (hunk 3 above)
Comment (codeboy5) [reply Kaggle/README.md]: Sure that would be beneficial, would make that change.

File: Kaggle/README.md (hunk 2 above)
Comment (codeboy5) [reply Kaggle/README.md]: **the model** would be more apt I believe

**Pull request closed without merging**